src/hmrc-content-guide/forms-publication/index.njk

```diff
@@ -6,47 +6,50 @@ layout: twoColumnPage.njk
 {% from "_codeSnippet.njk" import codeSnippet %}
 {% from "_example.njk" import example %}
 
-<h2 class="govuk-heading-l">Title</h2>
+<h2 class="govuk-heading-l">Document type</h2>
 
 <p class="govuk-body">
-Decide if the title should focus on the action a user needs to take or the name of the form. This will depend on how users will find and navigate to the form. 
+Publication: form (with a PDF, Open Document Format or external link attachment)
 </p>
 
-<p class="govuk-body">
-Do not include the form number.
-</p>
-
-<h2 class="govuk-heading-l">Summary</h2>
+<h2 class="govuk-heading-l">When to use</h2>
 
 <p class="govuk-body">
-If there’s a form number you should include it. For example:
+Before you use this pattern, you should <a href="https://design.tax.service.gov.uk/hmrc-content-guide/forms-check/" class="govuk-link">check which document type to use</a>, to decide
+if you should use a detailed guide instead.
 </p>
 
 <p class="govuk-body">
-'Register your business for VAT if you’re distance selling into Northern Ireland using form VAT1A'
+For an example see [Register for VAT if you&#39;re distance selling into Northern
```

Comment:
Think simplify to just...

For an example see [Register for VAT if you're distance selling into Northern Ireland](https://www.gov.uk/government/publications/vat-application-for-registration- distance-selling-vat1a).